README.md

```diff
@@ -4,6 +4,12 @@ In AWS IoT, a **stream** is a publicly addressable resource that is an abstracti
 
 More information about streams and MQTT based file delivery can be found [here](https://docs.aws.amazon.com/iot/latest/developerguide/mqtt-based-file-delivery.html)
 
+**AWS IoT Fleet Provisioning Library v1.1.0
```

Comment:
The name of the library needs to be corrected?

Reply:
Thanks for the catch. Update in new PR.